iceoryx2-cli/iox2-services/src/cli.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2024 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use clap::Parser;
+use clap::Subcommand;
+
+use iceoryx2_cli_utils::help_template;
+
+#[derive(Parser)]
+#[command(
+    name = "iox2-services",
+    about = "Query information about iceoryx2 services.",
+    long_about = None,
+    version = env!("CARGO_PKG_VERSION"),
+    disable_help_subcommand = true,
+    arg_required_else_help = false,
+    help_template = help_template("iox2-services", false),
+)]
+pub struct Cli {
+    #[clap(subcommand)]
+    pub action: Option<Action>,
+}
+
+#[derive(Parser)]
+pub struct DetailsOptions {
+    #[clap(long = "static", short = 's')]
```

Comment:
When you add here a comment with a description what this thing is good for then it also shows in `iox2 --help` (the proc macro gets it).

Especially for the `dynamic` member I would add a description since it is not directly obvious what this is good for.